src/dftbp/dftb/rshgamma.F90

```diff
@@ -206,8 +206,10 @@ function getHfAnalyticalGammaValue_workhorse(hubbu1, hubbu2, dist) result(gamma)
     !> Resulting gamma
     real(dp) :: gamma
 
-    real(dp) :: tauA, tauB
-    real(dp) :: tmp, tau
+    real(dp) :: tmp, tau, tauA, tauB, distTauA, invDist
+
+    distTauA = 0.0_dp
+    invDist = 0.0_dp
```

Comment:
Is there any reason to initialize those variables explicitly? (Or just a kind of "good practice" in order not to leave variables uninitialized?)

Reply:
Yes, "good practice" was my intention. I decided against an initialization at the declaration, because this implies the `save` attribute (at least as far as I understand), which might lead to some unexpected behavior.

Reply:
I have removed the added explicit initializations.